plugins/catalog-backend/api-report.md

```diff
@@ -165,6 +167,9 @@ export class CatalogBuilder {
 export type CatalogCollatorEntityTransformer =
   CatalogCollatorEntityTransformer_2;
 
+// @public (undocumented)
+export const catalogConflictsTopic = 'catalog.conflict';
```

Comment:
I think a repeating pattern in the backstage code is to put this in a file called "constants.ts" in the "src" directory, and use snake uppercase.